docs/runbook.md

```diff
@@ -41,15 +41,111 @@
 
 ### Configure Azure deployment
 
+Below section is for running cases on Azure platform, it specifies:
+  - admin_private_key_file: the private key file to access the Azure VM.
+  - subscription_id: Azure VM is created under this subscription.
+
+```yaml
+platform:
+  - type: azure
+    admin_private_key_file: $(admin_private_key_file)
+    azure:
+      subscription_id: $(subscription_id)
+```
+
 ### Select and set test cases
 
+Below section is to specify P0 and P1 test cases excluding case with name `hello`.
+
+```yaml
+testcase:
+  - criteria:
+      priority: [0, 1]
+  - criteria:
+      name: hello
+    select_action: exclude
+```
+
 ### Use variable and secrets
```

Comment:
This should be first one in the reference. And we don't need to write any variables in other examples.